src/index.html

```diff
@@ -15,8 +15,59 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li><a href="#">SAMSUNG</a></li>
+          <li><a href="#">SMARTPHONES</a></li>
+          <li>
+            <a
+              href="#"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li><a href="#">GADGETS</a></li>
+          <li><a href="#">TABLETS</a></li>
+          <li><a href="#">PHOTO</a></li>
+          <li><a href="#">VIDEO</a></li>
```

Comment:
Use semantic tags where possible. Instead of using `<li>` for navigation links, consider using `<nav>` combined with `<ul>` and `<li>` to define navigation lists semantically.